.golangci.yml

```diff
@@ -7,6 +7,7 @@ linters-settings:
 linters:
   enable-all: true
   disable:
+    - dupl
```

Comment:
I will fix this before I land. The lossy reading in Sender/Receiver is duplicated